Rakefile

```diff
@@ -41,10 +41,10 @@ class Installfest
     if boolean_expression
       return CommandResult.new(true, 'met')
     else
-      message = colorize("Expectation NOT met", :red)
+      message = "\nNOT met. Extra debugging info:"
       message += "\n  #{failure_message_for_actual}"
       message += "\n  #{failure_message_for_expected}"
-      return CommandResult.new(false, message)
+      return CommandResult.new(false, colorize(message, :grey))
```

Comment:
I'd prefer grey because students freak out when they see red. This way, with the error messages in light grey, we have the info if necessary, but students will just look to the green section below.